pkg/handlers/deploy.go

```diff
@@ -41,7 +41,6 @@ type UpdateUndeployResultRequest struct {
 type DeployAppVersionRequest struct {
 	IsSkipPreflights             bool `json:"isSkipPreflights"`
 	ContinueWithFailedPreflights bool `json:"continueWithFailedPreflights"`
-	IsAirgap                     bool `json:"isAirgap"`
```

Comment:
Will this now try to report in airgapped installs?

Reply:
Never mind, I see where this check moved

Reply:
it won't https://github.com/replicatedhq/kots/pull/1819/files#diff-706f7e57e6f7361d82c9a8f1b0c3ddb08c18e88d3ee384cf20964b0ca86254c2R60